docs/overview/why-saleor/extensibility.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Extensibility model
+---
+
+### API Only
+
+• **Webhooks.** You can subscribe to more than a hundred [events](/developer/extending/webhooks/overview) to receive data from Saleor.
+
+• **API extensions.** Saleor can [proxy](developer/extending/webhooks/synchronous-events/overview) calls to your service to modify behavior or resources.
+
+• **Dashboard extensions.** [Extend admin](/developer/extending/apps/overview) functionality using any front-end stack embeddable into _iframes_.
+
+• **Granular permission control.** [Granular permissions](/developer/permissions) for access tokens.
+
+• **GraphQL oriented.** Provide GraphQL queries to shape the data provided via Webhooks.
+
+### Why not plugin architecture?
+
+Plugin architecture is popular among open-source projects, classical examples would be Magento or Shopify.
+Such a model has direct coupling which the platform which can be convenient to get started quickly,
+but can also lead to many issues.
+
+**Benefits over plugin architecture:**
+
+• **Technology agnostic.**
+Technology stack freedom allowing you to pick the best tool for the job and access larger talent pool.
+
+• **Improved security.** Tightly coupling creates security risks.
+
+• **Independent deployment.** You can deploy your microservices independently.
+
+• **Reliability.** Less risk of degrading performance or bringing the whole system down.
+
+• **Scalability.** You can allocate resources more efficiently for your services.
+
+• **Multi-market.** Plugin architecture provides less independence between teams and marketers, creating higher chance of interference.
+
+• **Open Standards.**
+By using GraphQL we are able to tap larger ecosystem of tools and community rather using limited proprietary SDKs.
```

Comment:
Rewrite needed. The main point of this paragraph should be higher interoperability when building on open standards.

Proposal:
By building on API/Webhooks and open standards foundation you foster interoperability and broader community engagement, which contrasts with the proprietary SDKs often associated with plugin architectures.